nix-diff.cabal

```diff
@@ -17,25 +17,58 @@ cabal-version:       >= 1.10
 extra-source-files:  README.md
                      CHANGELOG.md
 
-executable nix-diff
-  main-is:             Main.hs
-  other-modules:
+library 
+  exposed-modules:
```

Comment:
If you expose these modules they should all begin with `Nix.Diff` to avoid conflicting with other packages